papers/vstte2024/llncs.cls

```diff
@@ -1,3 +1,6 @@
+# Copyright Kani Contributors
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
```

Comment:
We must not do that. That file has a copyright already.

Reply:
We could however add a modifications copyright. We would still need to add `%` as a [valid comment starter here](https://github.com/model-checking/kani/blob/6ac5183eabd704c04b88f80552bedf24f7c8e4e9/scripts/ci/copyright_check.py#L11-L19).

If we don't want to add a copyright statement, than @rahulku would need to add it to the [copyright-exclude file](https://github.com/model-checking/kani/blob/main/scripts/ci/copyright-exclude).